red/My-List-Util/lib/Test/My/List/Util.pm

```diff
@@ -0,0 +1,153 @@
+package Test::My::List::Util;
+
+use v5.16;
+use strict;
+use warnings;
+
+use Test::Builder;
+use Exporter 'import';
+our @EXPORT_OK = qw( sum_ok );
+
+my $Test = Test::Builder->new;
+
+=head1 NAME
+
+Test::My::List::Util - Utility for testing lists.
+
+=head1 VERSION
+
+Version 0.01
+
+=cut
+
+our $VERSION = '0.01';
+
+=head1 SYNOPSIS
+
+Utility functions for testing list operations.
+
+    use Test::My::List::Util;
+
+    sum_ok( sum( 1, 5, 3 ), 9, 'Simple sums work' );
+
+=head1 EXPORT
+
+=item sum_ok
+
+=back
+
+=head1 SUBROUTINES/METHODS
+
+=head2 sum_ok
+
+Verifies the outcome of a list summation is valid.
+
+=cut
+
+sub sum_ok {
+  my ( $actual, $expect, $name ) = @_;
+  $name //= "Sums to $expect";
+
+  if ( !defined $actual || $actual !~ /^-?\d*(?:\.\d+)?(?:e[+-]\d+)?$/ ) {
```

Comment:
Yup, I cover the ones I want to cover, but don't consider `nan` or `inf` to have equality (`inf` != `inf` in my books).